regression-test/suites/nereids_function_p0/load.groovy

```diff
@@ -27,6 +27,109 @@ suite("load") {
         DROP TABLE IF EXISTS `fn_test_bitmap`
     """
 
+    // test ipv4/ipv6
+    sql """ drop table if exists fn_test_ip_nullable """
+    sql """ CREATE TABLE IF NOT EXISTS fn_test_ip_nullable (id int, ip4 ipv4, ip6 ipv6) engine=olap
+                                                                                         DISTRIBUTED BY HASH(`id`) BUCKETS 4
+                                                                                         properties("replication_num" = "1") """
+    // make some special ip address
```

Comment:
It's very good to add comment for your test data. You can add comment for all data files.